src/track/text-style.js

```diff
@@ -157,7 +156,6 @@ class TextStyle {
   static toJson(text: TextStyle): PKTextStyleObject {
     return {
       fontEdge: text.fontEdge,
-      fontSize: text.fontSize,
       fontScale: text.fontScale,
```

Comment:
fontSize is referenced in api.md, so after fontSize is removed the docs should be rebuilt by running yarn run docs:generate